honeybee_energy/material/gas.py

```diff
@@ -34,7 +34,7 @@ class _EnergyWindowMaterialGasBase(_EnergyMaterialWindowBase):
                           'Xenon': (158.33970642, 0.0, 0.0)}
     MOLECULARWEIGHTS = {'Air': 28.97, 'Argon': 39.948,
                         'Krypton': 83.8, 'Xenon': 131.3}
-    __slots__ = ('_thickness',)
+    __slots__ = ('_thickness')
```

Comment:
This is going to break a lot of things since you turned the tuple into a single string. Please put the comma back before we merge.

Reply:
ooops grossly overlooked that.
fixed, commit: fixed accidental comma removal